modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -201,14 +201,16 @@ The syntax for array index configuration is shown below:
 |`path`
 |image:ROOT:no.png[]
 |Path to the array to be indexed.
+Use `"[]"` to represent a property that is an array of each nested array level.
```

Comment:
The double quotes are a little confusing here.
The monospace suffices for the square brackets - so are the double quotes then part of the expression (looking at `contacts[].phones`, I think not)?